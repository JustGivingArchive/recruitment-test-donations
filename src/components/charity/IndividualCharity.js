import React from 'react';

const IndividualCharity = ({ name, id, toggle, select }) => {
  return (
    <div onClick={() => select(id)} className="panel hoverable">
      <p className="paneltextlarge">{name}</p>
    </div>
  );
};

export default IndividualCharity;
