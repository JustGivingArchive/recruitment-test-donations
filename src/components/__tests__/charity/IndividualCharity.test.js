import React from 'react';
import renderer from 'react-test-renderer';
import IndividualCharity from '../../charity/IndividualCharity.js';

describe('Test charity info componenet', () => {
  const props = {
    name: 'name',
    id: 'id',
    select: () => null
  };

  it('renders correctly', () => {
    const tree = renderer.create(
      <IndividualCharity {...props} />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
