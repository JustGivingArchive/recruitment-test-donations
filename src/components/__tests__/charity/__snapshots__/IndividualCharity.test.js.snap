exports[`Test charity info componenet renders correctly 1`] = `
<div
  className="panel hoverable"
  onClick={[Function]}>
  <p
    className="paneltextlarge">
    name
  </p>
</div>
`;
